code/game/objects/items/weapons/tools.dm

```diff
@@ -721,6 +721,55 @@
 	playsound(get_turf(src), 'sound/medbot/Flatline_custom.ogg', 35)
 	return (SUICIDE_ACT_BRUTELOSS)
 
+/obj/item/tool/crowbar/halligan
+	name = "Halligan bar"
+	desc = "Combination pick, crowbar, and adze used for forcible entry."
+	icon = 'icons/obj/items.dmi'
+	icon_state = "halligan"
+	hitsound = "sound/weapons/toolhit.ogg"
+	item_state = "halligan"
+	w_class = W_CLASS_MEDIUM
+	attack_verb = list("pries", "slashes", "stabs", "bludgeons", "whacks")
+
+/obj/item/tool/crowbar/halligan/attackby(obj/item/I, mob/user)
+	if(istype(I,/obj/item/weapon/fireaxe))
+		var/obj/item/weapon/fireaxe/F = I
+		to_chat(user, "<span class='notice'>You attach the fireaxe and Halligan bar to carry them easier.</span>")
+		var/obj/item/tool/irons/SI = new (user.loc)
+		SI.fireaxe = F
+		SI.halligan = src
+		user.drop_item(F)
+		F.forceMove(SI)
+		user.drop_item(src)
+		forceMove(SI)
+		user.put_in_hands(SI)
+		return 1
+	return 0
+
+/obj/item/tool/irons
+	name = "set of irons"
+	desc = "Fireaxe and Halligan bar used for forcible entry."
+	icon = 'icons/obj/items.dmi'
+	icon_state = "irons"
+	hitsound = "sound/weapons/toolhit.ogg"
+	item_state = "irons"
+	w_class = W_CLASS_MEDIUM
+	force = 1.0
```

Comment:
Should combining the two tool tools really reduce the force that low?

Reply:
It's supposed to be very unwieldy so people only use this to carry the items instead of using it as a weapon. Carrying a fireaxe in this form removes the normal slowdown so it shouldn't be a viable weapon.

Reply:
Fair point, but I think it would at least be heavy enough to do damage akin to a toolbox if you smashed someone with it.

Reply:
I'll bring it up to 5 to mirror the crowbar's stats